README.md

```diff
@@ -194,13 +194,11 @@ ha_cluster_transport:
     - name: option2_name
       value: option2_value
   links:
-    -
-      - name: option1_name
+    - - name: option1_name
```

Comment:
ok - if this doesn't work, then just disable yamllint hyphen checking
```yaml
  -
    - name: option1_name  # yamllint disable-line rule:hyphens
```
that way when it is converted, it will come out like this:
```
  -   - name: option1_name  # yamllint disable-line rule:hyphens
```
and yamllint will not complain

Reply:
ok - looks like using `- - name: ...` didn't work - so use my suggestion

Reply:
done